mt-kahypar/partition/refinement/flows/parallel_construction.h

```diff
@@ -143,12 +143,12 @@ class ParallelConstruction {
                                       vec<HypernodeID>& whfc_to_node);
 
   // ! Only for testing
```

Comment:
Is this comment still accurate?

Reply:
The function still isn't supposed to be called from outside the class, so I would say yes